tests/internal/test_wrapping.py

```diff
@@ -95,6 +95,7 @@ def f(a, b, c=None):
     assert not channel1 and not channel2
 
 
+@pytest.mark.skipif(sys.version_info > (3, 12), reason="segfault on 3.13")
```

Comment:
We can't skip these I'm afraid. If we have segfaults it means we don't support 3.13 and need to fix (also the condition seems wrong 🤔 )

Reply:
```
tests/internal/test_wrapping.py::test_wrap_generator_yield_from Fatal Python error: Segmentation fault                                                                                                                                                                        
                                                                                                                                                                                                                                                                              
Current thread 0x0000ffffb4380020 (most recent call first):                                                                                                                                                                                                                   
  File "/root/project/tests/internal/test_wrapping.py", line 244 in g                                                                                                                                                                                                         
  File "/root/project/tests/internal/test_wrapping.py", line 248 in test_wrap_generator_yield_from                
```

Reply:
This sounds maybe related https://docs.python.org/3/whatsnew/3.13.html#cpython-bytecode-changes

Reply:
We decided in a different thread that these skips are ok to merge because no integrations use the functionality that these tests exercise.